docs/modules/ROOT/partials/getting-started/con-registry-artifact-metadata.adoc

```diff
@@ -0,0 +1,67 @@
+// Metadata created by nebel
+
+[id="registry-artifact-metadata"]
+= {registry} artifact metadata
+
+Whenever an item is added to {registry}, a set of metadata properties is stored along with the item content.  This
+metadata consists of a set of generated, read-only properties along with some properties that can be set by the user.
+
+.{registry} metadata properties
+[%header,cols=3*]
+|===
+|Property
+|Type
+|Editable
+|*id*
+a| string
+a| false
```

Comment:
Suggest using code format instead of bold for properties (e.g., `id` instead of *id*)